hack/make-rules/test-cmd.sh

```diff
@@ -103,7 +105,8 @@ setup
 run_kube_apiserver
 run_kube_controller_manager
 create_node
-SUPPORTED_RESOURCES=("*")
+# Array of all resources supported by the apiserver.
+export SUPPORTED_RESOURCES=("*")
```

Comment:
Same here, for `create_node`

Reply:
comment was added. similarly this may(?) be read https://cs.k8s.io/?q=SUPPORTED_RESOURCES&i=nope&files=&repos=kubernetes/kubernetes